packages/pigeon/pigeons/core_tests.dart

```diff
@@ -699,3 +699,460 @@ class TestMessage {
   // ignore: always_specify_types, strict_raw_type
   List? testList;
 }
+
+/// The core interface that each host language plugin must implement in
```

Comment:
This may be a case where we do actually want to make a second file instead of putting everything into the one giant file?